File: StadiumEngine/StadiumEngine.WebUI/ClientApp/src/models/dto/offers/LockerRoomDto.ts
import {LockerRoomGender} from "./enums/LockerRoomGender";
import {BaseEntityDto} from "../BaseEntityDto";

export interface LockerRoomDto extends BaseEntityDto {
    id: number,
    name: string,
    description: string,
    gender: LockerRoomGender,
    isActive: boolean,
}

File: StadiumEngine/StadiumEngine.DTO/Offers/LockerRooms/LockerRoomDto.cs
using StadiumEngine.Common.Enums.Offers;

namespace StadiumEngine.DTO.Offers.LockerRooms;

/// <summary>
/// ДТО раздевалки
/// </summary>
public class LockerRoomDto : BaseEntityDto
{
    /// <summary>
    /// Id
    /// </summary>
    public int Id { get; set; }

    /// <summary>
    ///  Название
    /// </summary>
    public string Name { get; set; }

    /// <summary>
    ///  Описание
    /// </summary>
    public string Description { get; set; }

    /// <summary>
    ///  Пол
    /// </summary>
    public LockerRoomGender Gender { get; set; }

    /// <summary>
    ///  Активность
    /// </summary>
    public bool IsActive { get; set; }
}